src/firebase/index.js

```diff
@@ -0,0 +1,18 @@
+import { initializeApp } from 'firebase/app'
+import { getAuth } from 'firebase/auth'
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyBNI-LIKJgqqNETBXDGNXw5dcRoYBiY1Xw',
+  authDomain: 'the-shire-eb558.firebaseapp.com',
+  projectId: 'the-shire-eb558',
+  storageBucket: 'the-shire-eb558.appspot.com',
+  messagingSenderId: '891665356224',
+  appId: '1:891665356224:web:edf45bac6411769b0a6425'
+}
```

Comment:
🔥
 When you need create keys, we don't let theses key public on the code!!! 
 For this case we use the `secrets` , in this case we can create theses `secrets` in `settings > security > secrets`

Reply:
Important ⚠️ :
in `DEV` environment, we using an `.env` file, that could exists only in this context.
in `QA`, `STG`, and `PROD`, we using `environment variables`.

https://www.youtube.com/watch?v=TLRW_xTnQwY&t=1781s